src/js/components/PageHeaderActionsMenu.js

```diff
@@ -0,0 +1,62 @@
+import {Dropdown} from 'reactjs-components';
+import React from 'react';
+
+import Icon from './Icon';
+
+const getMenuItems = (children) => {
+  return [
+    {
+      className: 'hidden',
+      html: <Icon id="ellipsis-vertical" size="mini" />,
+      id: 'trigger'
+    },
+    ...React.Children.map(children, getDropdownItemFromComponent)
+  ];
+};
+
+const handleItemSelection = (item) => {
+  if (item.clickHandler) {
+    item.clickHandler();
+  }
+};
+
+const getDropdownItemFromComponent = (child, index) => {
+  return {
+    clickHandler: child.props.clickHandler,
+    html: child,
+    id: index
+  };
+};
+
+const PageHeaderActionsMenu = (props) => {
+  return (
+    <Dropdown
+      buttonClassName="button button-link"
+      items={getMenuItems(props.children)}
+      onItemSelection={handleItemSelection}
+      persistentID="trigger"
+      transition={true}
+      dropdownMenuClassName="dropdown-menu"
+      dropdownMenuListClassName="dropdown-menu-list"
+      wrapperClassName="dropdown"
+      {...props} />
```

Comment:
Let's not pass all the props to the dropdown. We should only use this technique in very rare cases as it is really hard to track down those props.